ui/desktop/app/components/settings-card/index.hbs

```diff
@@ -0,0 +1,34 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<Hds::Card::Container @level='mid' @hasBorder={{true}}>
+
+  <div class='header-and-icon'>
+    <Hds::IconTile @icon={{@icon}} @size='small' />
+    <Hds::Text::Display @tag='h4'>
+      {{@header}}
+    </Hds::Text::Display>
+
+    {{#if @action}}
+      <div class='action'>
+        {{yield}}
```

Comment:
How's this work with the yield on line 33 as well?

Reply:
changed it to `@action` as in the next tickets, we will add action button in this block

Reply:
Just want to understand, wouldn't we still need to yield content? What would we be passing to `@action`?

Reply:
hmm, I didn't spend much time thinking this through, my initial thought was to pass in actions(signout/pause/restart) but that wouldn't work if there is more than one. now I'm thinking of passing in a named block to `@actions`?

Reply:
Yup makes sense, I think we'd have to do named blocks in this component

Reply:
updated!